docs/_interface/merge.md

```diff
@@ -48,25 +44,13 @@ For this two-client structure to work, consensus clients must be able to pass bu
 to be executed. Executing the transactions locally is how the client validates that the transactions do not
 violate any Ethereum rules and that the proposed update to Ethereum's state is correct. Likewise, when the node
 is selected to be a block producer the consensus client must be able to request bundles of transactions from Geth to
-include in the new block. This inter-client communication is handled by a local RPC connection using the 
-[`engine` API][engine-api-link] which is part of the JSON-RPC API exposed by Geth. 
-
-## Transition
-
-The transition from PoW to PoS will happen when a pre-announced total difficulty is reached by the chain. 
-This is different to other forks which usually occur at a certain scheduled block number. The total difficulty threshold 
-that will trigger the Merge is also known as the [*Terminal
-Total Difficulty* (TTD)](https://ethereum.org/en/glossary/#terminal-total-difficulty). In
-case of an emergency delay, the TTD can be overriden in Geth using the `--override.terminaltotaldifficulty` command-line
-flag. Once the merge block exists, Geth will no longer be able to follow the head of the chain without a consensus
-client. If Geth is not connected to a consensus client it will simply stall at the merge block. 
-Assuming a consensus client is connected in advance, the two clients will automatically
-handle the merge together with no disruption to the user.
+include in the new block and a resulting state change. This inter-client communication is handled by a local RPC 
```

Comment:
Maybe a nitpick, I feel like it's not very accurate. From Geth's perspective, CL needs to retrieve the execution layer block from Geth which not only includes the transaction set, but also a few other fields(e.g. stateRoot, logsBloom, etc). It's called `ExecutableData` in spec.

All in all, maybe we can call it `consensus client must be able to request a executable data which includes bundles of transactions and some other metadata from Geth to
include in the new block and a resulting state change`?

Reply:
ok - thanks